File: backend/src/main/java/com/tripfriend/domain/trip/information/dto/TripInformationUpdateReqDto.java
package com.tripfriend.domain.trip.information.dto;

import com.tripfriend.domain.trip.information.entity.Transportation;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TripInformationUpdateReqDto {

    private Long tripInformationId; // 여행 정보 ID

    private Long placeId; // 장소 ID

    private LocalDateTime visitTime;

    private Integer duration;

    private Transportation transportation;

    private int cost;

    private String notes;
}

File: backend/src/main/java/com/tripfriend/domain/trip/information/dto/TripInformationUpdateReqDto.kt
package com.tripfriend.domain.trip.information.dto

import com.tripfriend.domain.trip.information.entity.Transportation
import java.time.LocalDateTime

data class TripInformationUpdateReqDto(
    var tripInformationId: Long? = null, // 여행 정보 ID

    var placeId: Long,           // 장소 ID

    var visitTime: LocalDateTime,

    var duration: Int,

    var transportation: Transportation,

    var cost: Int = 0,

    var notes: String? = null,
)